metagraph-plugin/conda_recipe/meta.yaml

```diff
@@ -15,11 +15,14 @@ requirements:
   build:
     - git
     - cmake
+    - pytest
   host:
     - pip
     - python
+    - pytest
   run:
     - python
+    - pytest
```

Comment:
pytest is not needed here. The test/requires is enough. As written the user will have to install pytest to install our metagraph plugin which makes no sense.

Reply:
Keep the pytest in "run", but remove it in "host" and "build", right?

Reply:
No. It isn't needed in run. Normal running of this package doesn't require pytest. Only running the tests requires that, but they are not actually in the package anyway.

Reply:
Got it.